### PR TITLE
Pull request draft for porting

### PR DESCRIPTION
**Pull request by devgav**

### CODE REVIEW

**Comment by mhodgesatuh** [comment]
We need a dev-2.3 branch for this, once it passes codacity.  Master is feature frozen for the pending 2.2 release.

**Comment by fduckart** [comment]
Love this, but can you please 'squash' all your various commits into a single commit. One other thing... I see merge conflict markers still in at least one file.

**Comment by fduckart** [comment]
Looks awesome. It be great if you could squash all this into a single commit, eh.

**Comment by devgav** [comment]
After fixing codacy issues I will squash the extra commits

**Comment by mhodgesatuh** [comment]
Codacy checks are looking better.  Thoughts on addressing these?

Expected empty line before rule (rule-empty-line-before)
.btn-default {

Expected RBRACE at line 113, col 1.
>>>>>>> 85c66c12bc429fd095d4bec9de492bb66a54126c
--and there are more of these

Expected empty line before rule (rule-empty-line-before)
.manage-groupings td:last-child {

Expected empty line before rule (rule-empty-line-before)
.manage-groupings tr td {

Expected single space before "{" (block-opening-brace-space-before)
@mixin gradient($text-color: #ffffff, $bg-color: #6fa9be, $top-gradient: #7db1c4, $btm-gradient: #5a9cb4){

Expected single space before "{" (block-opening-brace-space-before)
@mixin btn-rmv ($bg-color: #b05b54, $top-gradient: #b05b54, $btm-gradient: #af534b){

Expected newline after "," (selector-list-comma-newline-after)
h1, h2, h3, h5, .text-left, span {

Unexpected unit (length-zero-no-unit)
padding: 0px;

Expected single space before "{" (block-opening-brace-space-before)
.font-weight-normal  {

Basically, you want to get as clean a report as possible.  In this case we are looking to at least address the easy ones.
